xml/admin_rbd.xml

```diff
@@ -1130,7 +1132,7 @@ EOF</screen>
      <command>feature enable</command> subcommand, the pool and image name, and
      the feature name:
     </para>
-<screen>&prompt.cephuser;rbd --cluster local feature enable <replaceable>POOL_NAME</replaceable>/<replaceable>IMAGE_NAME</replaceable> journaling</screen>
+<screen>&prompt.cephuser;rbd --cluster local feature enable <replaceable>POOL_NAME</replaceable>/<replaceable>IMAGE_NAME</replaceable> journal</screen>
```

Comment:
Problem I saw with this change is that the journal feature is different from the _journaling_ feature: https://docs.ceph.com/en/latest/rbd/rbd-mirroring/#enable-image-journaling-feature

Or at least that's how I understood it